doc/examples/denoise_patch2self.py

```diff
@@ -152,6 +152,10 @@
 save_nifti("denoised_patch2self.nii.gz", denoised_arr, affine)
 
 ###############################################################################
+# You can also use Patch2Self version 1 to denoise the data by using version
+# argument. The default version is set to 3. To use version 1, you can call
+# Patch2Self as follows:
+#    patch2self(data, bvals, version=1)
```

Comment:
This will not be displayed with an appropriate formatting.

Reply:
![image](https://github.com/user-attachments/assets/15b16862-8bf7-4e68-8787-170185523b2a)

It looks okay.

Reply:
My bad then. Sorry.